### DISCUSSION TITLE
Caub Question

### DISCUSSION THREAD

**Discussion by CJBrandi** [3kh0/ext-remover#1175] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So my district has a network that is managed by policy for all chromebooks to connect to. It has certain settings that can't be changed and one of them is metered network. As this setting is really the main thing behind caub I am confused. I usaully use another network instead of this one because it is firewall-less. Sometimes when my chromebook restarts it automatically connects to this network first. I don't believe caub is changing this metered setting on the chromebook network and I am worried my chromebook will enivetly update (before I get CRSH2TTY working or Icarus comes out in december). Is there any way to fix this or should I just hope nothing goes wrong? Please only answer if you know what you are talking about!

**Comment by joshuajohncohen** [comment, marked as answer]
Use https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/CAUB%20%2B%20DNS%20%2B%20Proxy%20Editor.html

**Reply by S-PScripts** [reply]
men

**Comment by schoolexploitkid** [comment]
what is icarus

**Reply by S-PScripts** [reply]
probably another chromeos bypass related to chromeshity
